product-management-adapters/src/test/groovy/io/dddbyexamples/factory/product/management/ProductDescriptionPersistenceSpec.groovy

```diff
@@ -1,11 +1,15 @@
 package io.dddbyexamples.factory.product.management
 
+import spock.lang.Ignore
+
 import org.springframework.beans.factory.annotation.Autowired
 import org.springframework.boot.test.context.SpringBootTest
 import spock.lang.Specification
 
 import static java.util.Collections.singletonList
 
+// TODO: Unignore
```

Comment:
PR that fixes this test is already merged.

Reply:
It doesn't work on my machine